src/style.css

```diff
@@ -1 +1,75 @@
-/* Styles go here */
+body {
+  margin: 0;
+
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+}
+
+.container {
```

Comment:
Use .header for this styles after you delete `<div class="container"></div>`

Reply:
To position element with selectors `.icon` and `.nav`, you just need to add padding to your `.header`